e2e/puppeteer_standalone.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+const puppeteer = require('puppeteer');
+const SERVER_URL = 'http://localhost:3001';
+
+const chromeOptions = {
+  executablePath:process.env.chrome,
+  headless: false
+};
+
+(async function main() {
+  try {
+    const browser = await puppeteer.launch(chromeOptions);
+    const page = await browser.newPage();
+    page.emulate({
+      viewport: {
+        width: 1280,
+        height: 1024
```

Comment:
```suggestion
        width: 1920,
        height: 1080
```
It's not 2004 anymore :)